_posts/2025-01-04-glorias-brink-epoch.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Celebrating Gloria's First Brink Epoch"
+permalink: /blog/2025/01/04/glorias-brink-epoch/
+layout: post
+author: brink
+name: "brink"
+image: /assets/images/gloria-zhao.jpg
+alt: Celebrating Gloria's First Brink Epoch
+category: "News"
+description: We’re delighted to celebrate Gloria Zhao's four year milestone of outstanding work on Bitcoin Core at Brink.
+---
+<div class="header-images">
+  <img src="/assets/images/gloria-zhao.jpg" alt="Gloria Zhao celebrates 4 years as a Brinkie">
+</div>
+
+Today, we’re delighted to mark a special milestone for Brink engineer Gloria
+Zhao: four years of outstanding work on Bitcoin Core at Brink. From her start as
+the very first Brink [fellow](/programs) to her key role as a
+Bitcoin Core maintainer and member of Brink’s grant committee, Gloria has
+pioneered significant advancements in Bitcoin’s development—and we couldn’t be
+more proud.
+
+## Taking a Chance on Bitcoin (and on Brink)
+
+Graduating from UC Berkeley in December 2020, Gloria had the pick of career
+paths including a coveted job offer to join Google full time after interning
+there the summer prior. Instead of taking the safe career path, she took a
+chance on Bitcoin development, and thanks to the visionary support of our
+[founding sponsors](/sponsors)—and early contributions from the
+[Human Rights Foundation and
+Spiral](/blog/2020/12/01/gloria/)—joined Brink as our first grantee.
+
+## A Package Relay Journey
+
+During Gloria's first year as a Brink fellow, she was mentored by Brink
+co-founder John Newbery and moved to London to establish our office there. After
+immersing herself in the Bitcoin Core codebase and coming up with her
+[fellowship project
+](/blog/2021/01/21/fellowship-project-package-accept/)in early
+2021, she was ready to put forth the project to the community with her [Package
+Mempool Accept and Package RBF
+proposal](https://gnusha.org/pi/bitcoindev/CAFXO6=+cHyQKM2n9yn4EhwLZO+AUB0ZD81qWPxmpN27rjUoU3w@mail.gmail.com/)
+in September of 2021.
+
+Commenting on the motivation for her work these past 4 years, Gloria notes:
+
+“*I work in a problem-oriented fashion, so all of the other mempool policy
+proposals stem from the core problem of fixing pinning attacks in exogenous fee
+bumping, which is a crucial tool in contracting protocols. Package Relay is
+maybe the “flagship” proposal and certainly the biggest chunk of work, but the
+name is often a catch-all for anti-pinning proposals including TRUC.*”
```

Comment:
(1) Move this to the top of the section
(2) Start with "Gloria is most known for her work in Package Relay and pinning attacks, which also chronicles the majority of her time at Brink. Commenting on the motivation for this long journey, Gloria notes:"

Reply:
Took this suggestion